targets/cmsis/TARGET_NORDIC/TARGET_MCU_NRF51822/TOOLCHAIN_GCC_ARM/TARGET_MCU_NRF51_16K_S110/NRF51822.ld

```diff
@@ -2,7 +2,7 @@
 
 MEMORY
 {
-  FLASH (rx) : ORIGIN = 0x00016000, LENGTH = 0x2A000
+  FLASH (rx) : ORIGIN = 0x00018000, LENGTH = 0x28000
```

Comment:
I think this was done because an overlapping soft device.
Is it possible that during testing a different soft device than yotta was used?
We need to double check that the same version of soft device is used in yotta and mbed-cli.

Reply:
@geowor01 please could you check

@microbit-carlos I've not got ARMCC6 to test this

Reply:
This was fixed upstream in https://github.com/ARMmbed/mbed-os/commit/dfcb2c6.
The armcc5 linker file already contains what we think might be the right value: https://github.com/lancaster-university/mbed-classic/blob/427f56863c9f9a9132a213ec583469dfc61c6244/targets/cmsis/TARGET_NORDIC/TARGET_MCU_NRF51822/TOOLCHAIN_ARM_STD/TARGET_MCU_NRF51_16K_S110/nRF51822.sct#L15
We still need to figure out which linker file yotta is using to make sure the micro:bit builds with Yotta and GCC are use the same address.